modules/segger/SEGGER_SYSVIEW_Conf.h

```diff
@@ -8,7 +8,7 @@
 #define SEGGER_SEGGER_SYSVIEW_CONF_H_
 
 #include <stdint.h>
-#include <zephyr/irq.h>
+#include <autoconf.h>
```

Comment:
`autoconf.h` should not be required, ref https://github.com/zephyrproject-rtos/zephyr/pull/44471

Reply:
Ah true, `autoconf.h` is "injected" automatically by the build system.